src/olympia/addons/tests/test_views.py

```diff
@@ -2802,6 +2802,27 @@ def test_exclude_addons(self):
         assert data['count'] == 1
         assert data['results'][0]['id'] == addon2.pk
 
+    def test_filter_fuzziness(self):
+        with self.activate('de'):
+            addon = addon_factory(slug='my-addon', name={
+                'de': 'Mein Taschenmesser'
+            }, default_locale='de')
+
+            # Won't get matched, we have a prefix length of 4 so that
+            # the first 4 characters are not analyzed for fuzziness
+            addon_factory(slug='my-addon2', name={
+                'de': u'Mein Hufrinnenmesser'
+            }, default_locale='de')
+
+        self.refresh()
+
+        with self.activate('de'):
+            data = self.perform_search(self.url, {'q': 'Taschenmssser'})
+
+        assert data['count'] == 1
+        assert len(data['results']) == 1
+        assert data['results'][0]['id'] == addon.pk
+
```

Comment:
This doesn't have to use any translations but I felt more confident finding German terms that are good test-candidates 😆